apps/home/stylesheets/hero.styl

```diff
@@ -1,38 +1,54 @@
+bp = 1024px
+
 .HomeHero
   &__focus
     position relative
     padding 0 2em
     font-size font-size-h2
     margin 2em auto
 
-  &__demo-block
+  &__content
     display flex
-    justify-content flex-start
-    width block_width
-
-    > .Block
-      margin 0 auto
+    width 100%
+    flex-direction row
+    justify-content center
+    align-items flex-start
+    flex 1
+    padding 6em 1em
+    +below(bp)
```

Comment:
After a moment I realize this means 'breakpoint' perhaps name it that?

Reply:
I shall